src/bics_bot/cogs/commands/intro_cmd.py

```diff
@@ -99,6 +132,35 @@ async def intro(
         # Add role to the user
         await user.add_roles(roles[year])
 
+        # Storing the user's birthday in JSON file
```

Comment:
same as the previous, this seems to be the same as birthday_cmd's code. organize these two blocks into two distinct functions inside `birthday_cmd.py`